_data/projects/Trapheus.yml

```
@@ -0,0 +1,13 @@
+name: Trapheus
+desc: Trapheus automates restoration of RDS database instances from snapshots into any dev, staging or production environments.It supports individual RDS Snapshot as well as cluster snapshot restore operations.
+site: https://github.com/intuit/Trapheus
+tags:
+  - rds
+  - step-functions
+  - python
+  - aws-lambda
+  - aws-lambda-layers
+  - database-restore
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/username/project/labels/up%20for%20grabs
```

This needs to be a URL that points to the list of issues labelled with `up for grabs`. If you agree that the label should be `good first issue` then we can update this too:

```suggestion
  link: https://github.com/intuit/Trapheus/labels/good%20first%20issue
```